src/api/handlers/testimonial.py

```diff
@@ -12,149 +12,156 @@
 
 class TestimonialHandler(RouteHandler):
 
-  def __init__(self):
-    super().__init__()
-    self.service = TestimonialService()
-    self.registerRoutes()
-
-  def registerRoutes(self):
-    self.add("/testimonials.info", self.info) 
-    self.add("/testimonials.create", self.create)
-    self.add("/testimonials.add", self.create)
-    self.add("/testimonials.list", self.list)
-    self.add("/testimonials.update", self.update)
-    self.add("/testimonials.delete", self.delete)
-    self.add("/testimonials.remove", self.delete)
-    self.add("/testimonials.rank", self.rank)
-
-    #admin routes
-    self.add("/testimonials.listForCommunityAdmin", self.community_admin_list)
-    self.add("/testimonials.listForSuperAdmin", self.super_admin_list)
-    
-
-  def info(self, request):
-    context: Context = request.context
-    args: dict = context.args
-
-    self.validator.expect("id", int, is_required=True)
-    self.validator.rename("testimonial_id", "id")
-    args, err = self.validator.verify(args)
-
-    if err:
```

Comment:
It seems you're reformatting the whole file by indenting with two more spaces.  Could you revert?  It makes it hard to nit pick what your real changes are.